ui/pages/providers/add.js

```diff
@@ -109,11 +180,38 @@ export default function ProvidersAddDetails() {
       return false
     }
 
-    router.replace('/providers')
+    if (!enableSCIM) {
+      router.replace('/providers')
+    }
 
     return false
   }
 
+  async function createSCIMAccessKey(id, name) {
+    try {
+      const res = await fetch('/api/access-keys', {
+        method: 'POST',
+        body: JSON.stringify({
+          userID: id,
+          name: name + '-scim',
+          ttl: '87600h',
+          extensionDeadline: '720h',
+        }),
+      })
+
+      const data = await res.json()
+
+      if (!res.ok) {
+        throw data
+      }
+
```

Comment:
I believe @ssoroka added global function `jsonBody` to handle this - so you can just do `await jsonBody(res)`

Reply:
Good catch, updated this to use this function